README.md

```diff
@@ -1 +1,33 @@
-# todo-list-peer-feedback
\ No newline at end of file
+# webpack-Todo list
+This a todo list project that uses webpack
+
+# Built With: 
+- HTML
+- CSS
+- JavaScript
+# Live Demo
+- https://natideme.github.io/webpack-demo
```

Comment:
- [ ] The live demo link of the project is not working as expected, kindly update the link to the live demo so that it can redirect the user immediately to the project